### DISCUSSION TITLE
Copy Obsidian Note GitHub Link easily

### DISCUSSION THREAD

**Discussion by paguasmar** [Vinzent03/obsidian-git#652] [Problems] [answered]
Problem:
I want to easily copy the GitHub link of an Obsidian file like I can in InteliJ by right clicking on a file. Right now to accomplish that feat I need to open GitHub on my browser, search for the Note repo, Search for the path of the note I need and then copy the link.

Why I need it:
I use Trello so it's handy to just past a GitHub link of a note I do not want to forget I did about that specific ticket.

**Comment by Vinzent03** [comment, marked as answer]
There are two existing commands to open the file on github. Just search for `github` in the command palette.

**Reply by paguasmar** [reply]
You are right. That's a much easier path. Thank you
